tests/acceptance/disabled-button-test.js

```diff
@@ -0,0 +1,51 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+import contains from '../helpers/contains';
+
+var App, DisabledController;
+
+module('Acceptance: Disabled AsyncButton', {
+  beforeEach: function() {
+    App = startApp();
+    DisabledController = App.__container__.lookup("controller:disabled");
+  },
+  afterEach: function() {
+    DisabledController.set('actionArgument1', undefined);
+    DisabledController.set('actionArgument2', undefined);
+    DisabledController.set('actionArgument3', undefined);
+    Ember.run(App, 'destroy');
+  }
+});
+
+    test('button works with custom disabled conditional', function(assert) {
```

Comment:
@danmcclain no worries. fixed. sorry as I messed up the formatting when rebasing.